package.json

```diff
@@ -84,5 +75,12 @@
         "lint": "eslint --cache clients/web Gruntfile.js grunt_tasks scripts clients/web/test && pug-lint clients/web/src/templates",
         "docs": "grunt docs",
         "esdoc": "esdoc"
+    },
+    "files": [
+        "clients/web/src/"
+    ],
+    "main": "clients/web/src/index.js",
```

Comment:
I'm thinking that this entry point should be in the `@girder/core` package.

This file should be essentially a super-build for the Girder "app", whereas `@girder/core` is the Girder library. So, this file's entry point should be `clients/web/src/main.js`.